webchat/components/header/ecis-header.component.js

```diff
@@ -20,5 +20,8 @@
     function headerController ($mdSidenav) {
         const headerCtrl = this;
 
+        headerCtrl.toggleNavbar = (componentId) => {
```

Comment:
create a service for changes in navbars
because you have the same function here and at contactsToolbarCtrl, and it'll be cloned in every place you need to toggle the navbar.